content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md

```diff
@@ -621,6 +622,38 @@ The Azure Artifacts password must be an unencoded token and should include a `:`
 
 You can check whether the private registry is successfully accessed by looking at the {% data variables.product.prodname_dependabot %} logs.
 
+### Cloudsmith
+
+For information about Cloudsmith and instructions on how to configure {% data variables.product.prodname_dependabot %} to work with Cloudsmith, see [Getting Started with Cloudsmith](https://help.cloudsmith.io/docs/welcome-to-cloudsmith-docs) and [Integrate Github Dependabot with Cloudsmith](https://help.cloudsmith.io/docs/dependabot), respectively.
+
+#### Upstreams to remote repositories
+
+Cloudsmith Upstreams proxy and cache dependencies into Cloudsmith from remote or public repositories. Instead of directly accessing a public repository (e.g., PyPI, Maven Central, Gradle, or NuGet), Cloudsmith proxies and caches the required dependencies. By caching packages from upstream sources, Cloudsmith ensures uninterrupted access to critical dependencies, mitigates risks associated with external service disruptions, optimizes package retrieval, and strengthens supply chain security. For more information, see [Upstreams](https://help.cloudsmith.io/docs/upstream-proxying-caching) in the Cloudsmith documentation.
+
+If the `replaces-base` setting is set to `true`, Dependabot will use the specified Cloudsmith URL as the primary source for dependencies instead of the default public repository for that package ecosystem. This means you should configure a corresponding Cloudsmith upstream to ensure Dependabot checks Cloudsmith first for dependencies. Here's an example configuration for Python:
+
+```yaml
+version: 2
+registries:
+  cloudsmith:
+    type: python-index
+    url: https://dl.cloudsmith.io/basic/YOUR-ORG/YOUR-REPO/python/
+    username: "${{ secrets.CLOUDSMITH_USER_NAME }}"
+    password: "${{ secrets.CLOUDSMITH_API_KEY }}"
+    replaces-base: true
+
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    registries:
+      - cloudsmith
+    commit-message:
+      prefix: "deps"
+    open-pull-requests-limit: 10
+```
+
```

Comment:
This seems like just a normal registry configuration for `pip`, no need to add an explicit example for it, I think if we just mention that Cloudsmith is supported that should be sufficient for users to figure out how to set it up.

Reply:
Will I make a new PR, just mentioning Cloudsmith?

Thanks again for reviewing this

Reply:
Closing this PR given the incoming new PR 💛